build/xtask/src/dist.rs

```diff
@@ -664,21 +660,17 @@ fn translate_elf_to_other_formats(
 }
 
 fn write_gdb_script(cfg: &PackageConfig, image_name: &str) -> Result<()> {
-    // Humility doesn't know about images right now. The gdb symbol file
```

Comment:
Can you please update the commit message to explain the motivation for this change? It is not clear to me what problem this change is trying to address? i.e. what didn't work before and how this change addresses that.

Reply:
Individual commit messages updated

Reply:
Thanks Richard! Looks good.